.circleci/config.yml

```diff
@@ -122,6 +122,9 @@ commands:
             develop)
               DEPLOY_ENV=dev
             ;;
+            DDP-5808-extend-file-upload-api)
```

Comment:
May be we can remove this before merge to develop

Reply:
Removed FB deploy settings from Circle CI config.